src/services/developer/developer.svc.coffee

```diff
@@ -0,0 +1,45 @@
+#=======================================
```

Comment:
There I see that the templates list is stubbed, but how do you handle the case where a developer tries to create/delete/update "new" widgets and receive an error from Mno API?
Is that stubbed somewhere else?